packages/contracts-bedrock/scripts/libraries/ForgeArtifacts.sol

```diff
@@ -37,11 +36,7 @@ library ForgeArtifacts {
     /// @notice Removes the semantic versioning from a contract name. The semver will exist if the contract is compiled
     /// more than once with different versions of the compiler.
     function _stripSemver(string memory _name) internal returns (string memory out_) {
-        out_ = Process.bash(
-            string.concat(
-                Executables.echo, " ", _name, " | ", Executables.sed, " -E 's/[.][0-9]+\\.[0-9]+\\.[0-9]+//g'"
-            )
-        );
+        out_ = Process.bash(string.concat("echo", " ", _name, " | ", "sed", " -E 's/[.][0-9]+\\.[0-9]+\\.[0-9]+//g'"));
```

Comment:
Multiple places in this file where we can just merge the strings together now

Reply:
`Process.bash(string.concat("echo", " ", _name, " | ", "sed", " -E 's/[.][0-9]+\\.[0-9]+\\.[0-9]+//g'"));`

becomes

`Process.bash(string.concat("echo ", _name, " | sed -E 's/[.][0-9]+\\.[0-9]+\\.[0-9]+//g'"));`

Reply:
done